level1/cmds/shell_21.asm

```diff
@@ -28,52 +26,45 @@
 tylg                set       Prgrm+Objct
 atrv                set       ReEnt+rev
 rev                 set       $00
-edition             set       22
+edition             set       23
```

Comment:
I can't tell if this is a pure comment update or if it has actual code changes that might break something and therefore needs an edition revision. Which is it?

Reply:
See my comment below.

Reply:
I would say if it's a pure (micro-)optimization that shouldn't be able to break anything, there's probably no need for a new edition.

But that's also why it's not great to make code and formatting (like comments) changes at the same time, because the tiny code update/fix/etc. that actually alters the binary gets hidden by the big line count changes that don't.

Reply:
I reverted the edition back to 22.